public/images/habitat-logo-by-chef.svg

```diff
@@ -1,11 +1,64 @@
-    </g>
-  </g>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This doesn't seem right. Try opening the file in Sketch, instead, then copying out the SVG code and comparing to the current version; the result should be pretty much just one `path`, similar to line 8 of the original.